subprojects/provider/src/test/kotlin/org/gradle/kotlin/dsl/caching/BuildCacheIntegrationTest.kt

```diff
@@ -91,15 +91,22 @@ class BuildCacheIntegrationTest : AbstractScriptCachingIntegrationTest() {
             assertThat(output, containsString(expectedOutput))
         }
 
-        // Cache hit from build cache
-        buildWithUniqueGradleHome("--build-cache", withBuildCacheIntegration).apply {
+        // Cache hit from build cache (enabled via gradle.properties file)
+        withUniqueGradleHome { gradleHome ->
 
-            compilationCache {
-                misses(cachedSettingsFile)
-                hits(cachedBuildFile)
-            }
+            File(gradleHome, "gradle.properties").writeText(
```

Comment:
:o: `gradleHome.resolve("gradle.properties")`?

Reply:
`gradleHome` is a `String`